govc/build.sh

```diff
@@ -8,6 +8,15 @@ git_version=$(git describe --dirty)
   exit 1
 fi
 
```

Comment:
build.sh should be deleted since we moved to goreleaser for building releases, you can just use go install: https://github.com/vmware/govmomi/blob/master/CONTRIBUTING.md#installing-from-source